web-app/js/portal/data/MetadataRecord.js

```diff
@@ -126,7 +126,10 @@ Portal.data.MetadataRecord = function() {
 
         var allowedOnlineResources = [];
 
-        Ext.each(Portal.app.appConfig.portal.metadataProtocols.webPage, function(protocol) {
+        Ext.each(Portal.app.appConfig.portal.metadataProtocols.supplementary, function(protocol) {
+            allowedOnlineResources.push(protocol.trim());
+        });
+        Ext.each(Portal.app.appConfig.portal.metadataProtocols.metadataRecord, function(protocol) {
```

Comment:
`_getPointOfTruthLink` in baseInjector.js uses `metadataRecord` 
`_getMetadataLink` calls `_getPointOfTruthLink`